snuba/query/query.py

```diff
@@ -5,6 +5,7 @@
     Any,
     Mapping,
     MutableMapping,
+    NamedTuple,
```

Comment:
🤔 

I'm guessing this got left here by accident?

Reply:
yes